ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_viirs.py

```diff
@@ -0,0 +1,470 @@
+    total_ob_processed = 0
+    for sat in unique_satids.tolist():
+        start_time = time.time()
+
+        matched = False
+        for satellite_info in satellite_info_array:
+            if (satellite_info["satellite_id"] == sat):
+                matched = True
+                satellite_id = satellite_info["satellite_id"]
+                satellite_name = satellite_info["satellite_name"]
+                satinst = sensor_name.lower()+'_'+satellite_name.lower()
+                logger.debug(f'Split data for {satinst} satid = {sat}')
+
+        if matched:
+
+            # Define a boolean mask to subset data from the original data object
+            mask = satid == sat
+            # MetaData
+            lon2 = lon[mask]
+            lat2 = lat[mask]
+            timestamp2 = timestamp[mask]
+            satid2 = satid[mask]
+            satzenang2 = satzenang[mask]
+            chanfreq2 = chanfreq[mask]
+            obstype2 = obstype[mask]
+            pressure2 = pressure[mask]
+            height2 = height[mask]
+            stnelev2 = stnelev[mask]
+
+            # Processing Center
+            ogce2 = ogce[mask]
+
+            # QC Info
+            qifn2 = qifn[mask]
+
+            # Method
+            swcm2 = swcm[mask]
+
+            # ObsValue
+            wdir2 = wdir[mask]
+            wspd2 = wspd[mask]
+            uob2 = uob[mask]
+            vob2 = vob[mask]
+
+            # Timestamp Range
+            timestamp2_min = datetime.fromtimestamp(timestamp2.min())
+            timestamp2_max = datetime.fromtimestamp(timestamp2.max())
+
+            # Check unique observation time
+            unique_timestamp2 = np.unique(timestamp2)
+            logger.debug(f'Processing output for satid {sat}')
+
+            # Create the dimensions
+            dims = {
+                'Location': np.arange(0, wdir2.shape[0])
+            }
+
+            # Create IODA ObsSpace
+            iodafile = f"{cycle_type}.t{hh}z.{data_type}.{satinst}.tm00.nc"
+            OUTPUT_PATH = os.path.join(ioda_dir, iodafile)
+            logger.info(f'Create output file : {OUTPUT_PATH}')
+            obsspace = ioda_ospace.ObsSpace(OUTPUT_PATH, mode='w', dim_dict=dims)
+
+            # Create Global attributes
+            logger.debug('Write global attributes')
+            obsspace.write_attr('Converter', converter)
+            obsspace.write_attr('sourceFiles', bufrfile)
+            obsspace.write_attr('dataProviderOrigin', data_provider)
+            obsspace.write_attr('description', data_description)
+            obsspace.write_attr('datetimeReference', reference_time)
+            obsspace.write_attr('datetimeRange', [str(timestamp2_min), str(timestamp2_max)])
+            obsspace.write_attr('sensor', sensor_id)
+            obsspace.write_attr('platform', satellite_id)
+            obsspace.write_attr('platformCommonName', satellite_name)
+            obsspace.write_attr('sensorCommonName', sensor_name)
```

Comment:
This attribute is not in the ObsSpace.yaml. I'd suggest adding it to the yaml though, as this information could be important.